web/src/muted_topics_ui.js

```diff
@@ -6,6 +6,7 @@ import * as channel from "./channel";
 import * as feedback_widget from "./feedback_widget";
 import {$t} from "./i18n";
 import * as message_lists from "./message_lists";
+import {handle_empty_narrow_banner} from "./narrow";
```

Comment:
You should follow the pattern of importing the module vs just the function here.